setup.bash

```diff
@@ -32,25 +34,26 @@ then
     apt-get install apt-utils  -y
 
     # Install Virtualenv
-    python -m pip install virtualenv
+    python3 -m pip install --upgrade pip
```

Comment:
You define the alias before for `python` to be version 3, so you don't need to use `python3`

Reply:
Yes we do need to use python3, because if we execute make deploy=false setup the alias will not be executed and by default pip is execute with python2 so using python3 we are ensure that the python3 is being used.

Reply:
But that will be in lines 49-57, right? (after the `else`) Here you defined the python alias so you want to use the actual alias you defined a few lines above?

Reply:
Yes, but sometimes when we don't specify python like python3, we end up to use python2 and this is the best way to ensure that python3 is been used.